packages/project-utils/packages/buildPackage.js

```diff
@@ -89,7 +94,12 @@ const babelCompile = async ({ cwd }) => {
 
         const paths = getDistFilePaths({ file, cwd });
         fs.mkdirSync(dirname(paths.code), { recursive: true });
+
+        // save the compiled JS file
         writes.push(fs.promises.writeFile(paths.code, code, "utf8"));
+
+        // save source maps file
+        // source map generation may be disabled, in this case we don't want to generate the file
```

Comment:
All looks good although this comment is now confusing. Says `source map generation may be disabled,`, but that's not true, isn't it? It's always `true`. 

Base on the `true` that we're now passing for source maps, does the `if` statement still make sense?

BTW please just use upper case for the first letter, when starting a sentence in comments. I also like to end the comment with `.`, but that's less important. Without it it's fine too (I've seen we have both styles in the codebase - both with the ending `.` and without it).

Reply:
That's true, this `if` is not relevant since I force-enabled source maps.